Blog/templates/Blog/user_detail.html

```diff
@@ -6,19 +6,22 @@
     <div class="row">
         <div class="cell-md-4 cell-sd-full">
             <div class="img-container profile-picture">
-                {% if user.profile.image %}
-                    <img src="{{ user.profile.image.url }}" style="border-radius: 50%"/>
+                {% if user_instance.profile.image %}
+                    <img src="{{ user_instance.profile.image.url }}" style="border-radius: 50%"/>
                 {% else %}
                     <img src="{% static 'images/default_user.png' %}"/>
                 {% endif %}
             </div>
-            <h1>{% trans 'Username' %}: {{ user }}</h1>
+            <h1>{% trans 'Username' %}: {{ user_instance }}</h1>
+        {% if user.is_authenticated %}
```

Comment:
The currently logged in user is available on the request: `request.user`. If you use this, you can revert the renaming of the context variable `user` to  `user_instance`.